.github/workflows/riot.yml

```diff
@@ -0,0 +1,27 @@
+name: RIOT examples
+
+on:
+  pull_request:
+
+jobs:
+  ci:
+    name: Build RIOT examples
+    runs-on: ubuntu-latest
+    container:
+      image: riot/riotbuild:latest
+    env:
+      RIOT_VERSION: "2024.07"
+      RIOTBASE: "/opt/riot-2024.07"
```

Comment:
This variable is needed by make to know where the RIOT sources can be found.
And since it is already there I reused it in the action to specify the path etc. to make it more durable